src/routes/safe/components/Balances/SendModal/index.jsx

```diff
@@ -1,35 +1,48 @@
 // @flow
 import CircularProgress from '@material-ui/core/CircularProgress'
-import { withStyles } from '@material-ui/core/styles'
+import { makeStyles } from '@material-ui/core/styles'
 import cn from 'classnames'
 import { List } from 'immutable'
 import React, { Suspense, useEffect, useState } from 'react'
 
 import Modal from '~/components/Modal'
 import { type Token } from '~/logic/tokens/store/model/token'
+import type { NFTToken } from '~/routes/safe/components/Balances/Collectibles/types'
+import ReviewCollectible from '~/routes/safe/components/Balances/SendModal/screens/ReviewCollectible'
+import SendCollectible from '~/routes/safe/components/Balances/SendModal/screens/SendCollectible'
 
 const ChooseTxType = React.lazy(() => import('./screens/ChooseTxType'))
 
 const SendFunds = React.lazy(() => import('./screens/SendFunds'))
 
+// const SendCollectible = React.lazy(() => import('./screens/SendCollectible'))
+
+// const ReviewCollectible = React.lazy(() => import('./screens/ReviewCollectible'))
```

Comment:
is this used?

Reply:
fixed in https://github.com/gnosis/safe-react/pull/570/commits/b8f57a91eb248230ea2728bf0447f2c0e1d0a493